### PR TITLE
CoreOS max payload

### PR DESCRIPTION
**Pull request by BillyAbildgaard**
creating max payload file for CoreOS.  My next story is to refine some options such as network devices by testing on real hardware.

@srinia6 @uppalk1 @tannoa2 @geoff-reid @dalebremner @keedya

### CODE REVIEW

**Comment by BillyAbildgaard** [comment]
test this please

File: example/samples/install_coreos_payload_full.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+    "options": {
+        "defaults": {
+            "hostname": "Coreos-node",
```
Comment (cgx027) [inline example/samples/install_coreos_payload_full.json]: change to lower case ```coreos-node``` seems more align with other examples :).

File: example/samples/install_coreos_payload_full.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            "installDisk": "/dev/sda",
+            "comport": "ttys0",
+            "repo": "http://172.31.128.1:9080/coreos/amd64-usr",
```
Comment (cgx027) [inline example/samples/install_coreos_payload_full.json]: I am not sure if change the host to a render-able ```{{ file.server }}/coreos/adm64-usr``` is a good idea. ignore me if it's not. @yyscamper

File: example/samples/install_coreos_payload_full.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            "rootPassword": "NotUsed",
+            "coreSshKeys": [ "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDJQ631/sw3D40h/6JfA+PFVy5Ofz6eu7caxbv0zdw4fTJsrFcOliHZTEtAvqx7Oa8gqSC6d7v61M0croQxtt1DGUcH2G4yhfdZOlK4pr5McwtX0T/APACdAr1HtP7Bt7u43mgHpUG4bHGc+NoY7cWCISkxl4apkYWbvcoJy/5bQn0uRgLuHUNXxK/XuLT5vG76xxY+1xRa5+OIoJ6l78nglNGrj2V+jH3+9yZxI43S9I3NOCl4BvX5Cp3CFMHyt80gk2yM1BJpQZZ4GHewkI/XOIFPU3rR5+toEYXHz7kzykZsqt1PtbaTwG3TX9GJI4C7aWyH9H+9Bt76vH/pLBIn rackhd@rackhd-demo"
+            ],
+            "users": [
+                {
+                    "name": "rackhd1",
+                    "password": "rackhd1",
```
Comment (cgx027) [inline example/samples/install_coreos_payload_full.json]: Seems using 123456 as password is more popular on other examples: Would be nicer to keeping consistency. 
https://github.com/RackHD/RackHD/blob/master/example/samples/install_esx_payload_full.json#L12
https://github.com/RackHD/RackHD/blob/master/example/samples/install_rhel_payload_full.json#L12

**Pull request merged**